### PR TITLE
[WebProfilerBundle] Added a conflict for Http-Kernel < 3.1

### PR DESCRIPTION
**Pull request by HeahDude**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/silexphp/Silex-WebProfiler/issues/94 |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by iltar** [comment]
Shouldn't this simply increase the minimum dependency?

File: src/Symfony/Bundle/WebProfilerBundle/composer.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
         "symfony/dependency-injection": "~2.8|~3.0",
         "symfony/stopwatch": "~2.8|~3.0"
     },
+    "conflict": {
+        "symfony/http-kernel": "<3.1"
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/composer.json]: You should then update the require section instead. And we should probably add a test if possible.

**Comment by HeahDude** [comment]
@iltar Of courses, I thought about it like it was in `require-dev`, thanks for noticing :)

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Pull request merged**